website/docs/components/dropdown/partials/code/component-api.md

```diff
@@ -62,68 +64,59 @@ The `Hds::Dropdown` component is composed of different child components, each wi
   </C.Property>
 </Doc::ComponentApi>
 
-### ListItem::CopyItem
+### ListItem::Interactive
 
 <Doc::ComponentApi as |C|>
-  <C.Property @name="copyItemTitle" @type="string"/>
   <C.Property @name="text" @required="true" @type="string">
-    The text to be copied. If no text value is defined, an error will be thrown.
+    Text to be used in the item. If no text value is defined, an error will be thrown.
   </C.Property>
-</Doc::ComponentApi>
-
-### ListItem::Description
-
-<Doc::ComponentApi as |C|>
-  <C.Property @name="text" @required="true" @type="string">
-    The text to be used for the description. If no text value is defined, an error will be thrown.
+  <C.Property @name="color" @type="enum" @values={{array "action" "critical" }} @default="action">
+    Acceptable values: “action” or “critical”.
+  </C.Property>
+  <C.Property @name="icon" @type="string">
+    Acceptable value: any [Helios icon](/foundations/icons/library) name.
+  </C.Property>
+  <C.Property @name="isLoading" @type="boolean" @values={{array "false" "true" }} @default="false">
+    Controls if the item is in “loading” state. When in this state, the item is not actually interactive, but you can pass the other expected arguments for the item (they’re simply ignored).
+  </C.Property>
+  <C.Property @name="href">
+    URL passed to the `<a>` element.
+  </C.Property>
+  <C.Property @name="isHrefExternal" @type="boolean" @values={{array "false" "true" }} @default="false">
+    Indicates whether or not the `<a>` link is external, in which case `target="_blank"` and `rel="noopener noreferrer"` attributes are added automatically.
+  </C.Property>
+  <C.Property @name="route models model query current-when replace">
+    Parameters passed as arguments to the `<LinkTo/LinkToExternal>` component.
+  </C.Property>
+  <C.Property @name="isRouteExternal" @type="boolean" @values={{array "false" "true" }} @default="false">
+    Controls if the "LinkTo" is external to the Ember engine ([more details here](https://ember-engines.com/docs/link-to-external)) in which case it will use a `<LinkToExternal>` instead of a simple `<LinkTo>` for the @route.
```

Comment:
```suggestion
    Controls if the “LinkTo” is external to the Ember engine ([more details here](https://ember-engines.com/docs/link-to-external)) in which case it will use a `<LinkToExternal>` instead of a simple `<LinkTo>` for the @route.
```

Reply:
You added curly quotes in other parts of the text so I suggested this change for consistency. 

Would you like me to check for these in general though? (I've been ignoring them in our text so far and only suggesting changes to use curly apostrophes since we seemed more agreed upon using those.)